src/schema/v2/system/time.ts

```diff
@@ -32,10 +32,26 @@ const SystemTimeType = new GraphQLObjectType<any, ResolverContext>({
 
 const SystemTime: GraphQLFieldConfig<any, ResolverContext> = {
   type: SystemTimeType,
-  description:
-    "Gravity system time, necessary for synchronizing device clocks.",
-  resolve: (_root, _options, { systemTimeLoader }) => {
-    return systemTimeLoader()
+  description: "Core system time, necessary for synchronizing device clocks.",
+  resolve: () => {
+    const now = new Date()
+    return {
+      day: now.getDate(),
+      wday: now.getDay(),
+      month: now.getMonth() + 1, // convert from 0-indexed to 1-indexed
+      year: now.getFullYear(),
+      hour: now.getHours(),
+      min: now.getMinutes(),
+      sec: now.getSeconds(),
+      dst: false, // TODO
+      unix: Math.floor(now.getTime() / 1000),
+      utcOffset: now.getTimezoneOffset() * -60,
+      zone: now
+        .toLocaleString("en", { timeZoneName: "short" })
+        .split(" ")
+        .pop(),
+      iso8601: now.toISOString(),
```

Comment:
A few of these transformations are simply trying to stay consistent with the old rails source.

We could _probably_ eliminate several of these (`dst`, `utcOffset`, `iso8601`). Only `unix` appears to be used. Thoughts?

Reply:
Any of these fields that aren't being used at all under `system { time` (from checking Force and Eigen), might as well get rid of them from the schema and drop resolving them! If they truly aren't being used that should be a seamless schema update (auto pushed out to clients and auto merged).

Reply:
Done 👍

Reply:
Cool - checking out the fields in the system endpoint in Gravity - these make sense to me as JS equivalents.